codecs/text/src/test/java/com/datastax/oss/dsbulk/codecs/text/string/StringToVectorCodecTest.java

```diff
@@ -53,39 +72,46 @@ void should_convert_from_valid_external() {
 
   @Test
   void should_convert_from_valid_internal() {
-    assertThat(dsbulkCodec)
+    assertThat(codec)
         .convertsFromInternal(vector)
-        .toExternal(vectorCodec.format(vector))
+        .toExternal(
+            "[1.1,2.2,3.3,4.4,5.5]") // this is NOT 100% identical to vector CQL representation
         .convertsFromInternal(null)
         .toExternal("NULL");
-
-    // We should encode
   }
 
   @Test
-  void should_not_convert_from_invalid_internal() {
-    assertThat(dsbulkCodec).cannotConvertFromInternal("not a valid vector");
+  void should_not_convert_from_invalid_external() {
+    assertThat(codec).cannotConvertFromExternal("[6.646329843]");
```

Comment:
This effectively winds up duplicating should_not_convert_too_much_precision() in a way that isn't very clear.  The original intent of this method was to perform something similar to JsonNodeToVectorCodecTest.should_not_convert_from_invalid_internal(), specifically given something that isn't a CqlVector this method fails completely.  We could certainly add a few more cases but I'd argue it's worthwhile to preserve the symmetry.